CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+### 24.9.9 [#786](https://github.com/openfisca/openfisca-core/pull/786)
+
+- Set a 120s default timeout in `openfisca serve`
+  - Otherwise, they regularly restart with an anxiety-provoking "[CRITICAL] WORKER TIMEOUT (pid:2137)" message in log
```

Comment:
What will happen instead now?

Reply:
They should now stay awake and no reboot for no reason